build/buildBuildImages.sh

```diff
@@ -158,14 +162,28 @@ function buildBuildScriptGeneratorImage() {
 }
 
 function buildGitHubActionsImage() {
+	local debianFlavor=$1
```

Comment:
maybe we could extend this function for not only github-actions use. Let it pass in an image tag(could be github-actions image, jamStack image or vso images), and return the buster based images accordingly.